### DISCUSSION TITLE
Remote in range to start recording

### DISCUSSION THREAD

**Discussion by ap0ll0nas** [gopro/labs#217] [Feature Requests]
I would like as soon as the remote is detected, recording to start. Use case is biking over a section of a trail. This is like an alternative to motion detection from far or with limited visibility. Or to avoid false alarms when others are in the scene. Same for stop record when remote is off range or when user stops via remote. Make this repeatable please.

Clarifying that recording should only start once a) the remote is out of range _and_ b) the remote goes in range, i.e. not just on b. Similarly for stopping, it should stop if a) the remote is in range and b) goes out of range. If the user starts or stops recording manually, via the remote or camera button, if the action is set to repeat it should continue - i.e. not like the current implementation of motion detection where pressing the record button stops it completely. If the user doesn't want this behavior to continue, there should be an option like pressing a button three times in a row, as is the case on other labs commands.

Ideally, this action would be combined with hindsight or motion detection or delayed start but the above would cover 90% of its use cases as is.

**Comment by dnewman-gpsw** [comment]
This is a cool idea, using the remote as a BLE range finder.

**Comment by dnewman-gpsw** [comment]
Here is it for HERO11 https://github.com/gopro/labs/discussions/283

**Comment by ap0ll0nas** [comment]
Hi David

I noticed that Gopro 12 doesn't offer HDR when the max lens mod 2.0 is attached/enabled. Is there a way to achieve this via labs ?

I also noticed that colors weren't as saturated as in Gopro 11, even if settings between those two were same and colors set to vivid. I just installed wide gamut on the 12 and was wondering if this has come up to your attention before.

Thanks
Apollon
________________________________

**Reply by dnewman-gpsw** [reply]
Hopefully HDR + MLM2 can be supported in the future, by it will not be through Labs. HDR is an advanced mode, not sure what the issues would be when combined with MLM2. Log is also not offered with MLM2, Labs does offer oMLOGB=400, for the same curve with MLM2.

Saturation, no idea, camera tuning is enhanced with each model. WIDE with flat or LOGB should be the same between models, as most of the tuning is bypassed.

**Comment by ap0ll0nas** [comment]
Thank you

________________________________
